docs/3.0.0-beta.x/concepts/controllers.md

```diff
@@ -260,6 +260,7 @@ module.exports = {
 };
 ```
 
-::: tip
-A route handler can only access the controllers defined in the `./api/**/controllers` folders.
+::: tips
+- A route handler can only access the controllers defined in the `./api/**/controllers` folders.
+- You must set permisions ([7. Set Roles and Permissions](https://strapi.io/documentation/v3.x/getting-started/quick-start.html#_7-set-roles-and-permissions)) for new routes before they are accessible to the client.
```

Comment:
Can you please use relative links to the .md files instead of hard coded html links?

Something like: `../getting-started/quick-start.md#_7-set-roles-and-permissions`

Reply:
Hello!
Thank you for the contribution but I cannot approve it.
Although I understand that it could be helpful to link here a documentation with more information about roles & permissions, we don't have such documentation yet. And I don't think it's a good idea to link to a section of the Quick Start.